backend/consensus/worker_service.py

```
@@ -509,20 +512,17 @@ async def health_check():
-                    timeout=aiohttp.ClientTimeout(total=timeout_s),
-                ) as resp:
-                    if resp.status != 200:
-                        _genvm_health_last_ok = False
-                        _genvm_health_last_error = (
-                            f"genvm_manager_status_http_{resp.status}"
-                        )
-                    else:
-                        _genvm_health_last_ok = True
-                        _genvm_health_last_error = None
-            except (aiohttp.ClientError, asyncio.TimeoutError) as exc:
+                resp = urlopen(Request(status_url), timeout=timeout_s)
```

Ruff (0.15.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)